parlai/core/agents.py

```diff
@@ -289,12 +289,13 @@ def load_agent_module(opt):
         # only override opts specified in 'override' dict
         if opt.get('override'):
             for k in opt['override']:
-                v = opt[k]
-                if str(v) != str(str(new_opt.get(k, None))):
-                    print("[ warning: overriding opt['" + str(k) + "'] to " +
-                          str(v) + " (previously:" +
-                          str(str(new_opt.get(k, None))) + ") ]")
-                new_opt[k] = v
+                if k in opt:
+                    v = opt[k]
+                    if str(v) != str(new_opt.get(k, None)):
+                        print("[ warning: overriding opt['{}'] to {} ("
+                              "previously: {} )]".format(
+                                k, v, new_opt.get(k, None)))
+                    new_opt[k] = v
```

Comment:
@emilydinan 

I tried:
```
py examples/display_model.py -t twitter -mf ~/tmp/models/twitter/twitter_seq2seq_model -bs 1 -bsrt False
```
and it didn't recognize the batch_sort flagged so it errored out